eventdiscovery-sdk/src/main/res/values/dimens.xml

```diff
@@ -13,4 +13,5 @@
     <dimen name="schedjoules_list_item_default_height">96dp</dimen>
     <dimen name="schedjoules_event_list_progress_bar">36dp</dimen>
     <dimen name="schedjoules_event_detail_item_icon">28dp</dimen>
+    <dimen name="schedjoules_toolbar_title">20sp</dimen>
```

Comment:
Please give this a more meaningful name. `schedjoules_toolbar_title` doesn't say anything. Is this the height, width, padding, margin, font size?
I suggest the following name scheme for dimensions: `schedjoules_<context>_<element>_<attribute>`, with each block written in camel-case. In this case this would result in `schedjoules_toolbar_title_fontSize` or (depending what you define as the "context": `schedjoules_eventList_toolbarTitle_fontSize`)
`schedjoules_event_list_progress_bar` would become `schedjoules_eventList_progressBar_width`.
What do you think?
Btw is it good practice to define the Toolbar title in `sp`? Isn't the Toolbar fixed to `64dp`? In theory, setting a title in `sp` could result in a title that is larger than the toolbar.